src/components/dialogs/network-modifications/voltage-level-topology-modification/voltage-level-topology-modification-dialog.tsx

```diff
@@ -0,0 +1,371 @@
+                        ? matchingSwitchInfos?.open
+                        : matchingAttributeEditData
+                          ? matchingAttributeEditData.equipmentAttributeValue
+                          : matchingSwitchInfos?.open;
+                    result.push({
+                        ...sw,
+                        type: SWITCH_TYPE,
+                        isModified: false,
+                        [CURRENT_CONNECTION_STATUS]: currentConnectionStatus,
+                    });
+                    const formValues = getValues(TOPOLOGY_MODIFICATION_TABLE);
+                    const index = formValues?.findIndex((item) => item.switchId === sw.switchId);
+                    if (index !== -1) {
+                        setValue(`topologyModificationTable.${index}.currentConnectionStatus`, currentConnectionStatus);
+                    }
+                });
+
+                if (unmodifiedSwitches.length > 0) {
+                    result.push({
+                        type: SEPARATOR_TYPE,
+                        id: 'unmodified-separator',
+                        title:
+                            intl.formatMessage({ id: 'unModifiedSwitchesSeparatorTitle' }) +
+                            ` (${unmodifiedSwitches.length})`,
+                        count: unmodifiedSwitches.length,
+                        [SWITCH_ID]: '',
+                        [PREV_CONNECTION_STATUS]: '',
+                        [CURRENT_CONNECTION_STATUS]: null,
+                    });
+
+                    unmodifiedSwitches.forEach((sw) => {
+                        result.push({
+                            ...sw,
+                            type: SWITCH_TYPE,
+                            isModified: false,
+                        });
+                    });
+                }
+            } else {
+                unmodifiedSwitches.forEach((sw) => {
+                    result.push({
+                        ...sw,
+                        type: SWITCH_TYPE,
+                        isModified: false,
+                    });
+                });
+            }
+            return result;
+        }
+        return [];
+    }, [
+        getValues,
+        isSwitchModified,
+        intl,
+        switchesToModify,
+        editData?.equipmentAttributeModificationList,
+        currentNode,
+        setValue,
+    ]);
+
+    return (
+        <CustomFormProvider
+            validationSchema={formSchema}
+            removeOptional={true}
+            {...formMethods}
+            isNodeBuilt={isNodeBuilt(currentNode)}
+            isUpdate={isUpdate}
+        >
+            <ModificationDialog
+                onClear={clear}
+                fullWidth
+                onSave={onSubmit}
+                maxWidth={'md'}
+                titleId="ModifyVoltageLevelTopology"
+                open={open}
+                keepMounted={true}
+                PaperProps={{
+                    sx: {
+                        height: '75vh',
```

Comment:
```suggestion
                        height: '95vh',
```
I have corrected the scrollbar, now you can fix size as 95vh as other modification if you want